README.md

```diff
@@ -12,104 +12,44 @@ npm install --save simplified-json
 
 ## Usage
 
-`sample.sjson`
```

Comment:
I'm not sure it's a bad idea to show an example sjson file for those who don't know what it looks like.  Perhaps we could trim it down or move it down, but are you sure we should just delete it?